tests/phpunit/tests/functions/isPhpVersionCompatible.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Tests the is_php_version_compatible function.
+ *
+ * @group functions.php
+ * @covers ::is_php_version_compatible
+ */
+class Tests_Functions_isPhpVersionCompatible extends WP_UnitTestCase {
+	/**
+	 * Provides test scenarios for all possible scenarios in wp_validate_boolean().
+	 *
+	 * @return array
+	 */
+	function data_provider() {
```

Comment:
```suggestion
	function data_php_version_compatible() {
```

It would be good to move this below the test if possible too, that's the convention followed throughout most of core